{{cookiecutter.__src_folder_name}}/root/docker-compose.yml

```diff
@@ -5,10 +5,10 @@ services:
     image: postgres:14
 
     environment:
-      DBSERVER_USER: postgres
-      DBSERVER_DB: relecloud
-      DBSERVER_HOST: db
-      DBSERVER_PORT: 5432
```

Comment:
This seemed to get the wrong env variable names at some point. Its also redundant with whats in the dev compose, so maybe could be rm'ed here or there.

Reply:
Let's discuss this early next week